src/components/Numbers/index.jsx

```diff
@@ -17,108 +17,92 @@ const Number = ({ number, title }) => {
+  const [errorMessage, setErrorMessage] = useState('');
 
-  componentDidMount() {
-    this._getData();
-  }
+  useEffect(() => {
+    _getData();
+  }, []);
 
-  _getData = () => {
-    this.setState({ loading: true });
+  const _getData = () => {
+    setLoading(true);
     charityAPI('/speaking-numbers')
       .then(({ data: { speaking_numbers, image_background } }) => {
-        this.setState({
-          loading: false,
-          data: {
-            numbers: speaking_numbers,
-            backgroundImage: image_background.url
-          },
-          error: false,
-          errorMessage: ''
+        setData({
+          numbers: speaking_numbers,
+          backgroundImage: image_background.url
         });
+        setLoading(false);
+        setError(false);
       })
       .catch(error => {
-        this.setState({
-          loading: false,
-          error: true,
-          errorMessage: 'can NOT fetch numbers'
-        });
+        setLoading(false);
+        setError(true);
+        setErrorMessage('can NOT fetch numbers');
       });
   };
 
-  render() {
-    const backgroundImageStyle = {
-      backgroundImage: `url(${this.state.data.backgroundImage})`
-    };
-
-    //while getting data
-    if (this.state.loading) {
-      return <div>loading...</div>;
-    }
+  const backgroundImageStyle = {
+    backgroundImage: `url(${data.backgroundImage})`
+  };
 
-    if (this.state.error) {
-      return (
-        <div>
-          {this.state.errorMessage}
-          {', '}
-          <a href="#/" className="text-c200 underline " onClick={this._getData}>
-            retry?
-          </a>
-        </div>
-      );
-    }
+  //while getting data
+  if (loading) {
+    return <div>loading...</div>;
+  }
 
-    if (!this.state.loading && !this.state.error) {
-      const {
-        data: { numbers }
-      } = this.state;
+  if (error) {
+    return (
+      <div>
+        {errorMessage}
+        {', '}
+        <a href="#/" className="text-c200 underline " onClick={this._getData}>
+          retry?
+        </a>
+      </div>
+    );
+  }
 
-      const numbersList = numbers.map(item => {
-        return <Number title={item.title} number={item.number} key={item.id} />;
-      });
-      return (
-        <section className="numbers z-0 relative bg-c800 pb-0">
-          <div className="container">
-            <div className="statistics-wrapper">
-              <div
-                className="statistics-wrapper__image bg-cover bg-no-repeat"
-                style={backgroundImageStyle}
-              ></div>
-              <div className="statistics-numbers">
-                <div className="statistics-numbers__speak relative text-center">
-                  <p className="statistics-numbers__speak__text p-4 font-normal absolute tracking-wide text-center capitalize text-lg font-body font-light text-c000 bg-c400">
-                    numbers speak
-                  </p>
-                </div>
+  if (!loading && !error) {
```

Comment:
this is is interesting :D

Reply:
i didn't add it .. the one how implement from static to react who add it :)
so i didn't edit it